docs/src/index.rst

```diff
@@ -12,106 +12,20 @@
-          `chargeable resources <http://docs.aws.amazon.com/general/latest/gr/glos-chap.html#chargeable-resources>`_,
-          such as running |EC2| instances or using |S3| storage.
-          Use the
-          `AWS Simple Monthly Calculator <http://calculator.s3.amazonaws.com/index.html>`_
-          to estimate charges for the use of various AWS resources.
-
-.. _additional_docs:
-
-Additional Documentation and Resources
-======================================
-
-In addition to this guide, the following are other resources available to |cdk| users:
-
-* `AWS Developer blog <https://aws.amazon.com/blogs/developer/>`_
-* `Gitter Channel <https://gitter.im/awslabs/aws-cdk>`_
-* `Stack Overflow <https://stackoverflow.com/questions/tagged/aws-cdk>`_
-* `GitHub repository <https://github.com/awslabs/aws-cdk>`_
-
-  * `License <https://github.com/awslabs/aws-cdk/blob/master/LICENSE>`_
-  * `Issues <https://github.com/awslabs/aws-cdk/issues>`_
-  * `Releases <https://github.com/awslabs/aws-cdk/releases>`_
-  * `Examples <https://github.com/awslabs/aws-cdk/tree/master/examples>`_
-  * `Documentation source <https://github.com/awslabs/aws-cdk/tree/master/docs/src>`_
-
-* `AWS CDK Sample for Cloud9 <https://docs.aws.amazon.com/cloud9/latest/user-guide/sample-cdk.html>`_
-* `AWS CloudFormation Concepts <https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-whatis-concepts.html>`_
-  
-.. _about_aws:
-
-About Amazon Web Services
-=========================
-
-Amazon Web Services (AWS) is a collection of digital infrastructure services that developers can
-leverage when developing their applications. The services include computing, storage, database, and
-application synchronization (messaging and queuing).
-
-AWS uses a pay-as-you-go service model. You are charged only for the services that you |mdash| or
-your applications |mdash| use. Also, to make AWS useful as a platform for prototyping and
-experimentation, AWS offers a free usage tier, in which services are free below a certain level of
-usage. For more information about AWS costs and the free usage tier, see
-`Test-Driving AWS in the Free Usage Tier <http://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/billing-free-tier.html>`_.
-
-To obtain an AWS account, go to `aws.amazon.com <https://aws.amazon.com>`_ and click :guilabel:`Create an AWS Account`.
+This documentation is the refernce for the |cdk|.
+The |cdk| documentation also includes the
+|cdk| User Guide.
+.. `AWS CDK User Guide <https://docs.aws.amazon.com/CDK/latest/userguide>`_.
 
 .. toctree::
    :maxdepth: 2
    :hidden:
 
-   Getting Started <getting-started>
-   Tutorial <tutorial>
-   Examples <examples>
-   Concepts <concepts>
-   AWS Construct Library <aws-construct-lib>
-   AWS CloudFormation Library <cloudformation>
-   Tools <tools>
-   Writing Constructs <writing-constructs>
+   Tool Reference <tool-ref>
```

Comment:
Let's just remove this from here and leave only the reference docs

Reply:
Done